projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -886,4 +905,64 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
   onTreeAction(row: any) {
     this.treeAction.emit({ row });
   }
+
+  allowDrop(event: DragEvent, dropRow) {
+    if (this.rowDraggable) {
```

Comment:
Isn't allowing rows to be dragged very different from dropping something (any kind of Element from outside the table) on a row? Aren't we mixing something here?

The idea is to have drag&drop support to be able to drop some other object from a different source onto a row. What does this have to do with whether the row is draggable or not?

Reply:
I assumed it to be internal dragg and drop. If we are also targeting external objects to be dropped then yes we need to change this.

Reply:
Now it can be used drop external objects.